### ISSUE TITLE
Specify rankdir

### ISSUE THREAD

**Issue by OliverUv** [pahen/madge#94]
Would like to be able to specify which rankdir should be used. The default (top to bottom) looks terrible for our project, while we get a very readable graph using 

```
digraph G {
  rankdir=LR;
  ...
```

**Comment by pahen** [comment]
Thanks for reporting! I'll look into this soon. Will probably be solved as a new config option `graphVizOptions` which will allow you to specify custom GraphViz options like `rankdir`.

**Comment by pahen** [comment]
This has been fixed in v1.1.0 now. Try adding this to your `.madgerc` config

```
{
	"graphVizOptions": {
		"G": {
			"rankdir": "LR"
		}
	}
}
```
